doc/conf.py

```diff
@@ -25,7 +26,8 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../'))
+
+sys.path.insert(0, os.path.abspath("../"))
```

Comment:
Shouldn't we go the same way as in OMC3 with the About file?
See https://github.com/pylhc/omc3/blob/85daa5e449ed0b5a87b6dc5eb91cd0e68be0d48e/doc/conf.py#L30
and following

Reply:
Implemented with a reading function instead of `exec`

Reply:
Done in 0d35cf4